temporal-sdk/src/main/java/io/temporal/internal/statemachines/WorkflowStateMachines.java

```diff
@@ -801,10 +802,10 @@ private void assertMatch(Command command, HistoryEvent event, Object expected, O
     public void workflowTaskStarted(
         long startedEventId, long currentTimeMillis, boolean nonProcessedWorkflowTask) {
       setCurrentTimeMillis(currentTimeMillis);
-      // If some new commands are pending and there are no more command events.
       for (CancellableCommand cancellableCommand : commands) {
+        // TODO delete this block
         if (cancellableCommand == null) {
-          break;
+          throw new Error("What the heck");
         }
```

Comment:
Looks like this guy should be removed :)

Reply:
Yeah, if all tests pass I will throw this away, this thing doesn't make much sense.